modules/grafana/templates/dashboards/publishing_api_overview.json.erb

```diff
@@ -829,6 +829,91 @@
         }
       ],
       "title": "New row"
+    },
+    {
+      "collapse": false,
+      "editable": true,
+      "height": "250px",
+      "panels": [
+        {
+          "aliasColors": {},
+          "bars": false,
+          "datasource": "Graphite",
+          "editable": true,
+          "error": false,
+          "fill": 1,
+          "grid": {
+            "threshold1": null,
```

Comment:
Ah, yep. You're correct. We'd have to do an explicit puppetdb lookup or export a resource to link them together. Which is a lot more work.